examples/mobile/bevy_mobile_example.xcodeproj/project.pbxproj

```diff
@@ -229,6 +229,7 @@
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
+				CODE_SIGN_IDENTITY = "";
```

Comment:
`CODE_SIGN_IDENTITY = ""` is needed to be able to build the app without signing it in CI

Reply:
Can you leave a comment in this type of file?

Reply:
The file is not meant to be read, it's generated and managed through xcode. It's also a setting specific to Bevy as an engine, someone creating a game and targeting iOS will have a signing identity. As Bevy is not paying the 100$/year apple development subscription, we don't have one.